dashboard/app/controllers/aichat_requests_controller.rb

```diff
@@ -97,6 +97,8 @@ def chat_request
   end
 
   private def can_access_aichat?
+    return true if current_user.levelbuilder?
+
     !DCDO.get("block_aichat_chat_completion", false) && current_user.has_aichat_access?
```

Comment:
I'm realizing that this check should already pass for levelbuilders because `teacher_can_access_ai_chat` checks `verified_instructor` which includes `levelbuilder`. Is the levelbuilder check here needed?

Reply:
Looks like it isn't needed; thanks.